contracts/handler/DefaultCallbackHandler.sol

```diff
@@ -48,4 +49,17 @@ contract DefaultCallbackHandler is ERC1155TokenReceiver, ERC777TokensRecipient,
         // We implement this for completeness, doesn't really have any value
     }
 
+    
+    function supportsInterface(bytes4 interfaceId) 
+        virtual 
+        override 
+        external 
+        view 
+        returns (bool) 
+    {
+        return interfaceId == type(ERC1155TokenReceiver).interfaceId
+            || interfaceId == type(ERC721TokenReceiver).interfaceId
+            || interfaceId == type(IERC165).interfaceId;
```

Comment:
So I guess this is the same that `interfaceId == this.supportsInterface.selector`

Reply:
I think so yes ... I just went with this as it was more in line we the statements above.